### ISSUE TITLE
auth-mailchimp-sync: Error Adding users into mailchimp

### ISSUE THREAD

**Issue by venkatabhilashchavala** [firebase/extensions#243]
Describe your configuration
Extension name: auth-Mailchimp-sync
Extension version: 0.1.0
Configuration values (redact info where appropriate):
Cloud Functions Locations: us-central1
Contact status Description: pending

**Describe the problem**
I was able to install the extension successfully.  I can see the adduserToList is executed successfully.  But not all the new users are imported into the MailChimp account.  Only one out of 4 new users are imported into my MailChimp account.  No errors in the logs.

logs:
ext-auth-mailchimp-sync-addUserToList
Function execution took 2150 ms, finished with status: 'ok' | more_vert
-- | --

**Steps to reproduce:**
I simply created a new blank list in Mailchimp, created a new API key for the account, and filled out the information in the configuration. I have a pre-existing Firebase Authentication instance running. The extension was installed successfully.
 
- Go to your Authentication dashboard.
- Click Add User to add a test user.

**Expected result**
All the newly added users in the firebase auth need to be added as contacts in my MailChimp account

**Actual result**
Only one out of 4 newly added users are imported into MailChimp.

**Comment by russellwheatley** [comment]
Hi @venkatabhilashchavala, I think it is because you have the `status` as `pending` which would indicate that they aren't subscribers (contacts) yet. I found this FAQ which appears to confirm this: 
<img width="923" alt="Screenshot 2020-04-07 at 13 40 51" src="https://user-images.githubusercontent.com/16018629/78670278-6a98fe00-78d5-11ea-9588-f8c0a47793b0.png">
https://mailchimp.com/help/troubleshoot-mailchimp-subscribe/

Although, it is interesting that they are appearing one in four times as subscribed. When i tried with a `pending` status, no users appeared in my subscriber panel. They all appeared when I set the status to `subscribed`.
